vaadin/info.yaml

```
@@ -0,0 +1,4 @@
+url: https://github.com/vaadin/quarkus
+issues:
+  repo: quarkusio/quarkus
```

Create the tracking issue in your own repository (see https://github.com/quarkusio/quarkus-ecosystem-ci#what-if-my-repository-is-not-in-the-quarkiverse-organization): 
```suggestion
  repo: vaadin/quarkus
```